src/Http/WebUtilities/src/FileMultipartSection.cs

```diff
@@ -19,7 +19,7 @@ public class FileMultipartSection
         /// </summary>
         /// <param name="section">The section from which to create the <see cref="FileMultipartSection"/></param>
         /// <remarks>Reparses the content disposition header</remarks>
-        public FileMultipartSection(MultipartSection section)
+        public FileMultipartSection(MultipartPipeSection section)
```

Comment:
This would be a breaking change, either need a new type or make the breaking change (I prefer the former if people are using these types).

Reply:
This is an option, but I am not sure how to use them both in FormFeature - currently FormFeature only uses the new PipeReader implementation - the biggest question is how Users actually use FileMultipartSection - as far as I know when using MVC (API or Razor) you use IFormFile. Of course if this class is used externally there is probably no reason to create a breaking change.